config/jetstream.php

```diff
@@ -78,4 +78,4 @@
 
     'profile_photo_disk' => 'public',
 
-];
+];
```

Comment:
Github doesn't like files without a new line at the end of the file. Can you please add a new line here?